templates/default/resource.rb.erb

```diff
@@ -1 +1,9 @@
-# To learn more about Custom Resources, see https://docs.chef.io/custom_resources/
+resource_name :<%= cookbook_name.gsub('-', '_') %>_<%= recipe_name %>
+provides :<%= cookbook_name.gsub('-', '_') %>_<%= recipe_name %>
+default_action :create
+
+property :foo, String, default: 'Example property.'
```

Comment:
Do we want to create a fake property like that that we'll have to change anyway? I think it might be better to just remove this.

Reply:
I thought it might be useful to have a reference for the property syntax.

Reply:
Might be better to leave that as a comment then and add a ``# TODO: example below`` and maybe also include a URL to the docs that provide more info.